src/main/java/pages/OrderConfirmationPage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class OrderConfirmationPage {
+    // Locators:
+    public By orderConfirmationMessageLocator = By.xpath("//h1[contains(text(), 'YOUR ORDER HAS BEEN PROCESSED')]");
+    public By invoicePageHyperlinkLocator = By.xpath("//div[@id='maincontainer']//a[contains(@href,'rt=account/invoice')]");
+    public By contactPageHyperlinkLocator = By.xpath("//div[@id='maincontainer']//a[contains(@href,'rt=content/contact')]");
+    public By continueButtonLocator = By.xpath("//div[@id='maincontainer']//a[@title='Continue']");
```

Comment:
I don't recommend searching for xpaths by text.

Reply:
for ``invoicePageHyperlinkLocator`` and ``contactPageHyperlinkLocator`` try:
- ``//div[@class='contentpanel']//a[1]``
- ``//div[@class='contentpanel']//a[2]``

Locators related to the order of elements in the DOM structure can also be unstable but certainly better than link content

Reply:
@Fishgame1 As discussed on the call, locators were amended to be more stable